arches/app/views/api.py

```diff
@@ -1555,14 +1555,14 @@ def get(self, request, resourceid):
 
         if "cards" not in exclude:
             readable_nodegroup_ids = [
-                nodegroup.pk
-                for nodegroup in get_nodegroups_by_perm(
+                nodegroup_id
+                for nodegroup_id in get_nodegroups_by_perm(
```

Comment:
This unrelated change (and sim. below) should trickle up from 7.6 soon. Just here so the new unit test that incidentally hits these lines doesn't fail.